local.conf.appendix

```diff
@@ -1,3 +1,20 @@
+# Enable testing
 
+# Needed since a bbappend in meta-ivi adds libusb PACKAGECONFIG which is 
+# broken for some weird reason
+BBMASK .= "|meta-ivi/recipes-devtools/qemu/"
```

Comment:
Maybe this should be done not only when testing? So moved to the meta-pelux/variant/common/local.conf?

Reply:
@tobsan what do you think?

Reply:
If it is a generic setting that we always want to have enabled, then it should of course go in ``meta-pelux/variant/common/local.conf``

Reply:
Furthermore, shouldn't there be a change upstream to meta-ivi that fixes this `PACKAGECONFIG`? I guess it would be nice to inform them about this. Does the downstream project experience these side effects as well?

Reply:
@GordanM You were removing a bunch of qemu stuff. This is still ok to include in our general config?

Reply:
I guess there are two topics being discussed now. The upstreaming and solving of the qemu devtool from meta-ivi and if we should include this change in the local.conf now.

Reply:
@thenor I've removed some meta-bistro QEMU startup scripts that are not being used anywhere, should not impact any functionality. 

Nice! That is good to know. We can always create two separate changes for this, maybe this even belongs in the temp patches then? You know best, I've just wanted to point this out :).